### ISSUE TITLE
Limbo fails at runtime while loading libqemu-system-x86_64

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#8]
```
What steps will reproduce the problem?
1. Start VM
2. Run logcat
3. Limbo fails to load .so lib because it cant find LOG_TMR

What is the expected output? What do you see instead?
Should find symbol and load lib succesfully.

Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 4 Oct 2012 at 2:04

**Comment by GoogleCodeExporter** [comment]
```
Workaround:
Add the following line temporarily at the end of stdio.h and recompile (Another 
ugly workaround but works for now)
#include logutils.h
```

Original comment by `ideal2i...@gmail.com` on 4 Oct 2012 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
A patch fix provided by Yair Lifshitz has been applied for this issue. This is 
planned for release 0.9.3.
```

Original comment by `ideal2i...@gmail.com` on 28 Oct 2012 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
Patch by Yair approved, fix will be included in rel 0.9.3.
```

Original comment by `ideal2i...@gmail.com` on 28 Oct 2012 at 7:37
- Changed state: **Fixed**
